src/lib/model/contract.ts

```diff
@@ -16,10 +17,15 @@ interface InstantiatedByMeState {
 
 export const useInstantiatedByMe = (enable: boolean): InstantiatedByMeState => {
   const { address } = useCurrentChain();
-  const { data: contracts = [], isLoading } = useInstantiatedListByUserQuery(
-    enable ? address : undefined
+  const isFullTier = useTierConfig() === "full";
+
+  const fullData = useInstantiatedListByAddress(address, enable && isFullTier);
+  const liteData = useInstantiatedContractsByAddressLcd(
```

Comment:
fullData -> resApi
liteData -> resLcd

we can change to other wording, but need to change other changed files too for consistency